bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Flight/Maneuver.cpp

```diff
@@ -0,0 +1,183 @@
+/// Apache License 2.0
+
+#include <OpenSpaceToolkit/Astrodynamics/Flight/Maneuver.hpp>
+
+inline void OpenSpaceToolkitAstrodynamicsPy_Flight_Maneuver(pybind11::module& aModule)
+{
+    using namespace pybind11;
+
+    using ostk::mathematics::curvefitting::Interpolator;
+
+    using ostk::astrodynamics::flight::Maneuver;
+
+    class_<Maneuver>(
+        aModule,
+        "Maneuver",
+        R"doc(
+            Spacecraft Maneuver class.
+            Store an acceleration and mass flow rate profile of a spacecraft maneuver.
+        )doc"
+    )
+
+        .def(
+            init<const Array<Instant>&, const Array<Vector3d>&, const Shared<const Frame>&, const Array<Real>&>(),
+            arg("instants"),
+            arg("acceleration_profile"),
+            arg("frame"),
+            arg("mass_flow_rate_profile"),
+            R"doc(
+                Constructor.
+
+                Args:
+                    instants (list[Instant]): An array of instants, must be sorted.
+                    acceleration_profile (list[numpy.ndarray]): An acceleration profile of the maneuver, one numpy.ndarray per instant in m/s^2.
+                    frame (Frame): A frame in which the acceleration profile is defined.
+                    mass_flow_rate_profile (list[float]):  A mass flow rate profile of the maneuver, one float per instant in kg/s.
+            )doc"
+        )
+
+        .def(self == self)
+        .def(self != self)
+
+        .def("__str__", &(shiftToString<Maneuver>))
+        .def("__repr__", &(shiftToString<Maneuver>))
+
+        .def(
+            "is_defined",
+            &Maneuver::isDefined,
+            R"doc(
+                Check if the maneuver is defined.
+
+                Returns:
+                    bool: True if the maneuver is defined, False otherwise. (Always returns true).
```

Comment:
@vishwa2710 I wasn't actually planning on changing that in the future. My idea with the maneuver object, is that since its just a holder of maneuver information, it can't be constructed with partial information, therefore once it is constructed it will always be defined. What do you think about this?